### DISCUSSION TITLE
Catalan translations

### DISCUSSION THREAD

**Discussion by Arthur-Barreto** [fastapi/fastapi#12951] [Translations]
Catalan translations tracking 🎉

We can use this Issue to track the progress of the Catalan language translations for FastAPI documentation.

Please let us know here what page you plan on translating so we can coordinate effectively. 🚀

Tips
We chose to use "tu" instead of more formal variations like "vostè" or "vosaltres" to maintain consistency and a conversational tone.
If the term is a technical term, leave it in English and add an explanation of the meaning in Catalan. For example:
Use "instància" for "instance", while keeping the original term in context for clarity. You can achieve this using notes or abbreviations (e.g., <abbr>).

Thank you for your help! Gràcies per la teva ajuda! 🎉

**Comment by svlandeg** [comment]
Because the automation wasn't triggered yet, posting this manually:

There's a new translation PR to be reviewed: https://github.com/fastapi/fastapi/pull/12952 by @Arthur-Barreto. 🎉 This requires 2 approvals from native speakers to be merged. 🤓

**Comment by YuriiMotov** [comment]
:loudspeaker: **Update on Translations**

We're updating the way we handle documentation translations.

Until now, we invited community members to translate pages via pull requests, which were then reviewed by at least two native speakers. While this has helped bring FastAPI to many more users, we’ve also run into several challenges - some languages have only a few translated pages, others are outdated and hard to maintain over time.
To improve this, we’re working on automation tools 🤖 to manage translations more efficiently. Once ready, documentation will be machine-translated and still reviewed by at least two native speakers ✅ before publishing. This will allow us to keep translations up-to-date while reducing the review burden on maintainers.

What’s changing now:

* 🚫 We’re no longer accepting new community-submitted translation PRs.

* ⏳ Existing open PRs will be reviewed and can still be merged if completed within the next 3 weeks (since July 11 2025).

* 🌐 In the future, we will only support languages where at least three active native speakers are available to review and maintain translations.

This transition will help us keep translations more consistent and timely while better supporting our contributors 🙌. Thank you to everyone who has contributed so far - your help has been invaluable! 💖
